### ISSUE TITLE
:cli_options set by Mix task are ignored.

### ISSUE THREAD

**Issue by smn** [cjbottaro/faktory_worker_ex#34]
The result is that any values provided as `-q` or `--queues` are ignored, `mix faktory` will only listen on the `default` queue as a result.

**Comment by smn** [comment]
A work around is:

```elixir
defmodule MyApp.FaktoryWorker do
  use Faktory.Worker, otp_app: :my_app

  def init(config) do
    config
    |> Keyword.merge(Faktory.get_env(:cli_options))
  end
end
```

**Comment by cjbottaro** [comment]
Fixed in 2842b62905e4b7a4f66a78e582d045da6740e281 and new release cut https://hex.pm/packages/faktory_worker_ex/0.7.1

If the new version works for you, can you close the issue? Thanks for reporting!

**Comment by smn** [comment]
Sorry forgot to report back, yes, it works :) Thank you!
